server/models/views/station.js

```diff
@@ -272,9 +272,13 @@ class ViewModel {
     let thresholds = []
 
     if (this.station.recentValue && !this.station.recentValue.err) {
+      let tVal = this.station.recentValue._.toFixed(2)
```

Comment:
This might be better as a ternary on line 277, it will remove the need to use `var`